docs/reference/run.md

```diff
@@ -901,7 +902,13 @@ remote server at `192.168.0.42` on port `123`
 
     $ docker run --log-driver=syslog --log-opt syslog-address=tcp://192.168.0.42:123
 
-`syslog-tag` specifies tag for syslog messages from container.
+`syslog-facility` specifies the syslog facility. This can be an integer of 0 to 23 or any of the
```

Comment:
This is really hard to read without the list.  Also, even though we provide bad examples earlier, should't be starting a sentence with the lowercase command.  

--> https://gist.github.com/moxiegirl/abe4816bb529908fae92

The `syslog-facility` option configures the syslog facility. By default, the system uses the `daemon` value. To override this behavior, you can provide an integer of 0 to 23 or any of the following named facilities: 

* `kern`
* `user`
* `mail`
* `daemon`
* `auth`
* `syslog`
* `lpr`
* `news`
* `uucp`
* `cron`
* `authpriv`
* `ftp`
* `local0`
* `local1`
* `local2`
* `local3`
* `local4`
* `local5`
* `local6`
* `local7`

The `syslog-tag` specifies tag for syslog messages from container. By default, the system uses the first 12 characters of the container id. To override this behavior, specify a `syslog-tag` option